### PR TITLE
Code cleaned

### PR DESCRIPTION
**Pull request by jwong28**
I tried to get the healthbar but it didn't work so I'm going to switch to enemies

### CODE REVIEW

**Comment by jwong28** [comment]
ALso: 
- Changelog updated
- Removed an unnecessary comment
- Switched a few lines of code

@AryanJ-NYC

File: Stickdom/level1.js [hunk 1]
```diff
@@ -254,6 +245,7 @@ class level1 extends Phaser.Scene
                 this.bulletTwo.setGravityY(35);
                 this.bulletNum = 1;
             }
+            console.log(this.player.x);
```
Comment (AryanJ-NYC) [inline Stickdom/level1.js]: No `console.log()` in master.

File: Stickdom/level1.js [hunk 2]
```diff
@@ -60,15 +60,15 @@ class level1 extends Phaser.Scene
...
         this.player.body.setGravityY(300) 
         //direction player facing
-        var direction = "right";
+        var direction;
         direction = "left";
         //Player collision with platform
         this.physics.add.collider(this.player, platforms);
         //Camera follows player
         this.cameras.main.startFollow(this.player); 
+        console.log(this.cameras.cameras[0]);
```
Comment (AryanJ-NYC) [inline Stickdom/level1.js]: No `console.log()` in master.

**Comment by AryanJ-NYC** [comment]
@jwong28 This should be merged first.

**Comment by jwong28** [comment]
@AryanJ-NYC 

- Basic health bar implemented
- Will implement a health decrease when hit in a different feature branch

**Comment by AryanJ-NYC** [comment]
Breaking: Player gets hit by ball and user cannot move.

**Comment by jwong28** [comment]
- Healthbar now changes when getting hit by bomb
- Bomb colliding with player destroys the bomb
- Readme updated

**Comment by AryanJ-NYC** [comment]
Nicely done. Adding issues relating to this but not bad so far.

**Pull request merged**